### PR TITLE
Format NEWS as Markdown

### PR DESCRIPTION
**Pull request by phw**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [x] Other
* **Describe this change in 1-2 sentences**:

# Problem

Currently the NEWS.txt uses some rather arbitrary formatting. Converting it to markdown would give it some better defined structure and would allow us to use a proper Markdown parser in places where we make use of this data, e.g. for https://picard.musicbrainz.org/changelog/

If we want to go that way we also need to update picard-website. I can work on that if we decide to do the Markdown conversion.

* JIRA ticket (_optional_): [PICARD-XXX](https://tickets.metabrainz.org/browse/PICARD-XXX)

# Solution

# Action

### CODE REVIEW

**Comment by zas** [comment]
I'm ok with the move to markdown, it will need few changes around, but historic NEWS.txt isn't good anymore for sure.
Though i'd like actual links in markdown instead of just [PICARD-XXX]

**Comment by phw** [comment]
@zas: I linked the ticket numbers. It of course makes the plain file a bit hareder to read, but the positive part is the rendered version is much nicer.

Also adding this to the homepage will simplify things a lot. Instead of our current regex approach we can just use a Markdown parser.

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,1163 @@
...
+- [PICARD-1012](https://tickets.metabrainz.org/browse/PICARD-1012) - Buttons on the "User Interface" and "Scripting" pages are smaller than buttons in other places
+- [PICARD-1016](https://tickets.metabrainz.org/browse/PICARD-1016) - Multiple images in related tracks confusing
+- [PICARD-1021](https://tickets.metabrainz.org/browse/PICARD-1021) - Picard loads all pending files before quitting
+- [PICARD-1024](https://tickets.metabrainz.org/browse/PICARD-1024) - Allow specifying a configuration file path
+- [PICARD-1030](https://tickets.metabrainz.org/browse/PICARD-1030) - Allow to add/replace cover art images and keep existing cover art
+
+
+# Version 1.4 - 2017-02-14
+## Bug-fix
+- AcoustID submission fails with code 299 ([PICARD-82](https://tickets.metabrainz.org/browse/PICARD-82))
```
Comment (zas) [inline NEWS.md]: It would be great to have those formatted as new entries

Comment (phw) [reply NEWS.md]: Good point, did some more regexing and updated the old entries.

**Comment by phw** [comment]
I think it's looking good now: https://github.com/metabrainz/picard/blob/9aaa38eec1d5a7c30b69771b4ac462a5163ea887/NEWS.md

**Pull request merged**